package.json

```diff
@@ -50,11 +50,11 @@
     "lint:eslint": "eslint './core/**/*.js' 'dev/**/*.js' gui/main.js gui/ports.js 'gui/js/**/*.js' './test/**/*.js'",
     "mocha": "env-cmd .env.test electron-mocha ./test/support/logging.js",
     "mocha:coverage": "yarn mocha --require ./test/support/coverage --reporter ./test/support/istanbul_reporter",
-    "postinstall": "yarn postinstall:electron && yarn postinstall:elm",
-    "postinstall:electron": "electron-builder install-app-deps",
-    "postinstall:elm": "elm-package install -y",
+    "install:all": "yarn install:electron && yarn install:elm",
+    "install:electron": "electron-builder install-app-deps",
+    "install:elm": "elm-package install -y",
     "prebuild": "yarn clean",
-    "reinstall": "bash -c 'rm -rf node_modules/' && yarn",
+    "reinstall": "bash -c 'rm -rf node_modules/' && yarn && yarn install:all",
```

Comment:
This was useful when CLI was a separate package, but do we really still need it?

Reply:
I don't know, I don't use the `reinstall` target but I don't want to break it for others

Reply:
@aenario Do you still use it? (I believe I didn't use it once since we dropped the separate cli/gui packages)